var/spack/repos/builtin/packages/mfem/package.py

```diff
@@ -254,6 +255,10 @@ class Mfem(Package, CudaPackage, ROCmPackage):
     conflicts("+slepc", when="~petsc")
     conflicts("+pumi", when="~mpi")
     conflicts("timer=mpi", when="~mpi")
+    conflicts("~lapack", when="+mumps")
+    conflicts("~metis", when="+mumps")
+    conflicts("~mpi", when="+mumps")
+    conflicts("mfem~openmp ^mumps+openmp")
```

Comment:
The first and second of these conflicts should be removed -- these are dependencies of MUMPS that do not need to be forced on MFEM.

The third one makes sense, though I'd rewrite it as other conflicts above: `conflicts("+mumps", when="~mpi")`.

Regarding the last conflict: I'm not sure it is strictly needed -- can't MFEM just call MUMPS which will then use OpenMP?

Are conflicts 1, 2, and 4, added just to simplify linking with MUMPS?

Reply:
It was just to simplify linking with MUMPS. I changed the conflicts like you suggested, and added the openmp flags similar to how it's done for the strumpack variant, and it still works.